src/test/java/com/tom/page/object/Excercise01Page.java

```diff
@@ -0,0 +1,61 @@
+package com.tom.page.object;
+
+import com.tom.driver.manager.DriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static java.lang.Integer.parseInt;
+
+public class Excercise01Page {
+
+    @FindBy(css = ".input-group > input.form-control")
+    private List<WebElement> inputNumberOfItems;
+
+    @FindBy(css = ".input-group-btn > button")
+    private List<WebElement> buttonAddItemToCart;
+
+    @FindBy(css = ".caption p:nth-child(2)")
+    private List<WebElement> priceOfOneItem;
+
+    @FindBy(className = "summary-quantity")
+    private WebElement summaryNumberOfItemsInCart;
+
+    @FindBy(className = "summary-price")
+    private WebElement summaryPriceOfItemsInCart;
+
+    private By buttonRemoveFromCart = By.cssSelector("[data-remove-from-basket]");
```

Comment:
use private final when you are doing inline initialization and you never want to change field reference to other object